### PR TITLE
socialhub-svg:1.0.0

### PR DESCRIPTION
**Pull request by Bi0T1N**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: Explain what the package does and why it's useful.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [ ] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by reknih** [comment]
Thank you for your submission!

The problems we have to address are largely IP-related:

Although `socialhub` would be an acceptable name on its own, I found active trademarks on it in advertising (Nice 38) in the US, China, and Italy, as far as I can see, each by a different company. There is also the very similar Socialhub+ trademark for digital services (Nice 42) in China. Lastly, there is a [German company](https://socialhub.io) that uses that name in trade. All in all, I think that this package exposes you and us to too much risk and should be renamed.

The other point is on the license: You license the package as MIT, but it has to be made clear somewhere (best place is the README) that the logos are the property of their respective trademark holders and that you are neither officially endorsed by them nor claim any ownership over the marks.

File: packages/preview/socialhub-svg/typst.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[package]
+name = "socialhub-svg"
+version = "1.0.0"
+repository = "https://github.com/Bi0T1N/typst-socialhub-svg"
+entrypoint = "socialhub-svg.typ"
+authors = ["Nico Neumann (Bi0T1N)"]
+license = "MIT"
+description = "A Typst library for Social Media references with scalable vector graphics icons."
+keywords = ["social", "media", "icons", "svg"]
+exclude = ["examples"]
```
Comment (reknih) [inline packages/preview/socialhub-svg/typst.toml:10]: Ah, and I noticed that you could add the `categories = ["components"]` key to make your package more discoverable.

**Comment by Bi0T1N** [comment]
> Although `socialhub` would be an acceptable name on its own, I found active trademarks on it in advertising (Nice 38) in the US, China, and Italy, as far as I can see, each by a different company. There is also the very similar Socialhub+ trademark for digital services (Nice 42) in China. Lastly, there is a [German company](https://socialhub.io) that uses that name in trade. All in all, I think that this package exposes you and us to too much risk and should be renamed.

I just used the name from my other PR at #218 but you're correct if there are some trademarks it's better to rename it and using `socialhub` was a bad idea. I'll come up with another name in the next days.

> The other point is on the license: You license the package as MIT, but it has to be made clear somewhere (best place is the README) that the logos are the property of their respective trademark holders and that you are neither officially endorsed by them nor claim any ownership over the marks.

Thanks, I'll also add and source for every SVG icon. Better safe than sorry.

**Comment by Bi0T1N** [comment]
What about `sociallist` ,`sociallistings`, `social-listings` (all short: `sl`) or `socialmedialistings` (short: `sml`)? Would one of them be accepted?

**Comment by reknih** [comment]
All of these names are rather descriptive - they are not in the spirit of our naming rules. Given that I approved `socialhub` in 218 I am willing to let it slide. However, we will remove both packages if we receive a complaint from one of the rights holders of the name.

I would recommend that you rename both packages in a future update.

**Comment by Bi0T1N** [comment]
Well, it's kinda hard to come up with a name that isn't descriptive as these icons are very special and have a single purpose. A landlord won't use them for his monthly rental invoice but maybe for his own CV/project showcase (exception: you're the trademark holder and writing your letters in Typst).
But names containing `trademark` or `icons` will also be to descriptive or too general (`trademarkhub` will also cause law problems). And I don't intend to have icons for every website in the world - just the big ones you might want to use in your CV.

**Comment by JKRhb** [comment]
Not sure if it is relevant for this version anymore, but how about a name like `socio` or `iconic`?

**Comment by Bi0T1N** [comment]
> Not sure if it is relevant for this version anymore, but how about a name like `socio` or `iconic`?

Thanks - these are good suggestions! I like `iconic` :+1: 
This motivated me to look for an acronym: `salmon` for SociAL Media icONs

@reknih: Would it be okay to use one of these two? Maybe even combined: `iconic-salmon` (that's what I'd prefer! :sunglasses:)

**Comment by laurmaedje** [comment]
@Bi0T1N Any of those three would be fine.

**Comment by Bi0T1N** [comment]
I opened a new PR #536 with the adjusted `typst.toml`, new name and a trademark disclaimer.

**Pull request closed without merging**